modules/profile.php

```diff
@@ -0,0 +1,15 @@
+<?php
+if (!$_SESSION["login_user_id"]) {
+    header('Location:index.php');
+}
+?>
+
+<!-- MAIN content -->
+<div id="main">
+    <div id="main-content">
+        <h3>My profile.</h3>
+        <p>Name: Lorem Ipsum</p>
```

Comment:
Please check Homework No.04 again, it required to show logged in user information on this page.